rts/lra/coordinator-quarkus/src/main/resources/application.properties

```diff
@@ -4,3 +4,6 @@ quarkus.ssl.native=true
 
 # allow Swagger in prod mode
 quarkus.swagger-ui.always-include=true
+
+quarkus.index-dependency.mp-lra.group-id=org.eclipse.microprofile.lra
```

Comment:
The team is discussing the removal of the quarkus runner for the coordinator since it doesn't provide enough added value. If users want a runner for a dockerised version of the coordinator then we will provide a WildFly based container.

The other changes in this PR look reasonable.